rclcpp/include/rclcpp/subscription_options.hpp

```diff
@@ -75,6 +75,42 @@ struct SubscriptionOptionsBase
   TopicStatisticsOptions topic_stats_options;
 
   QosOverridingOptions qos_overriding_options;
+
+  /// Set use_default_callbacks.
+  RCLCPP_PUBLIC
+  SubscriptionOptionsBase &
+  set_use_default_callbacks(bool uses)
+  {
+    use_default_callbacks = uses;
+    return *this;
+  }
```

Comment:
Sorry I missed this before, but we really try to avoid implementation in headers, in order to avoid ODR violations and also to decrease compile times.

Since this class is not templated, I'd say it should have a `.cpp` file (it didn't have one before because it didn't have any methods) and these should be implemented there.